tuned/plugins/plugin_scheduler.py

```diff
@@ -135,7 +135,7 @@ def get_priority_max(self, sched):
 		return schedutils.get_priority_max(sched)
 
 class SchedulerPlugin(base.Plugin):
-	"""
+	r"""
```

Comment:
With this change you also accidentally fixed another issue :slightly_smiling_face:.

In multiple places, the docstring contains `\b` which escaped to backspace characters and caused the docs to display incorrectly when calling `get_plugin_documentation`. Of course, this doesn't happen with raw strings.